lib/JiraClient.ts

```diff
@@ -0,0 +1,51 @@
+import fetch = require("node-fetch");
```

Comment:
Why not like this? To keep it consistent with `OctokitAction`?
```suggestion
import fetch from 'node-fetch';
```

Reply:
I trusted the tool, but it failed me 😢